executor/show.go

```diff
@@ -304,6 +304,17 @@ func (e *ShowExec) fetchShowColumns() error {
 		}
 
 		desc := table.NewColDesc(col)
+		var columnDefault interface{}
```

Comment:
var columnDefault string

Reply:
hi @crazycs520, IMHO it's better to keep `columnDefault` as `interface{}` for `nil` values, thanks!